<?xml version="1.0"?>
<launch>

    <node name="light_node" pkg="cr_control" type="light_node" output="screen"/>

</launch>